lib/pure/times.nim

```diff
@@ -158,7 +158,9 @@ when defined(posix):
 
   type CTime = posix.Time
 
-  var CLOCK_REALTIME {.importc: "CLOCK_REALTIME", header: "<time.h>".}: Clockid
+  var
+    realTimeClockId {.importc: "CLOCK_REALTIME", header: "<time.h>".}: Clockid
+    cpuClockId {.importc: "CLOCK_THREAD_CPUTIME_ID", header: "<time.h>".}: Clockid
```

Comment:
Can we keep the names the same as the ones in C? Adding our own names is just mental overhead in this case imo

Reply:
I chose a logical name for the CPU clock id because different POSIXes might want or need different clocks, and it is easier to change the definition of the logical id that it is to change the proc body that calls `clock_gettime`.

Reply:
OK, that's fair